### PR TITLE
Loading message in python

### PR DESCRIPTION
**Pull request by shaurya-blip**

### CODE REVIEW

**Comment by shaurya-blip** [comment]
What is wrong with this?

File: loader.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""
+Shaurya Pratap Singh 
+@shaurya-blip
+
+Shows loading message while doing something.
+"""
+
+import itertools
```
Comment (bupboi1337) [inline loader.py:8]: Always list dependencies in a requirements.txt file!

File: loader.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import threading
+import time
+import sys
+
+# The task is not done right now
+done = False
+
+
+def animate(message="loading", endmessage="Done!"):
```
Comment (bupboi1337) [inline loader.py:17]: For smaller file size I would replace message with start and endmessage with end

**Review by bupboi1337 — APPROVED** [review]
Seems like it would be a amazing addition to this repo.

**Comment by bupboi1337** [comment]
> What is wrong with this?

Lint-python fails when it thinks it finds mistakes in your code. I would check out the log for it and fix what caused it to fail.

**Comment by shaurya-blip** [comment]
Found anything yet?

**Pull request merged**